pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.smartbear</groupId>
     <artifactId>ready-jira-plugin</artifactId>
-    <version>1.2</version>
```

Comment:
This is a not only one place. Also, please change version in https://github.com/SmartBear/ready-jira-plugin/blob/master/src/main/java/com/smartbear/ready/plugin/jira/PluginConfig.java

Reply:
thank you.